lib/tasks/lit_tasks.rake

```diff
@@ -26,4 +36,9 @@ namespace :lit do
       end
     end
   end
+
+  desc 'Remove all translations'
+  task clear: :environment do
+    Lit::LocalizationKey.destroy_all
```

Comment:
it should also destroy `Lit:: IncommingLocalization` and call `Lit.init.cache.reset'

Reply:
:+1:  will do